sbi/inference/potentials/score_based_potential.py

```diff
@@ -79,41 +78,36 @@ def set_x(
         self,
         x_o: Optional[Tensor],
         x_is_iid: Optional[bool] = False,
-        rebuild_flow: Optional[bool] = True,
+        atol: float = 1e-5,
```

Comment:
Okay, so now we always rebuild a flow when we set `x`. And any time we call a method from `ScorePosterior`, it always sets `x` first, so the flow always gets rebuilt. when we either `set_default_x` manually, or when we call any method from `ScorePosterior`. Am I understanding correctly?

Reply:
Yes! whenever we set a new `x`, the flow is rebuild. 
when calling the potential, it will just use the current flow and there is no option to rebuild it with every call.

Reply:
Great, this is a good solution!